src/raytracer/Raytracer.m

```diff
@@ -125,7 +125,17 @@
 polarizationTx = [1, 0];
 polarizationRx = [1, 0];
 
-MaterialLibrary = importMaterialLibrary('raytracer/Material_library.txt');
+% Define Material library for various environments    
+switch paraCfgInput.inputScenarioName
+    case 'LectureRoom'
+            MaterialLibrary = importMaterialLibrary('raytracer/Material_library_LectureRoom.txt');
+    case 'ParkingLot'
+            MaterialLibrary = importMaterialLibrary('raytracer/Material_library_ParkingLot.txt');
```

Comment:
i don't see this one

Reply:
I removed parking lot since we are working on it.

Reply:
then please remove any reference to it

Reply:
Right now,I commented parking lot and data center lines.

Reply:
removing is preferred over commenting when the code get published. it's really ugly to publish code with commented portions here and there